File: Semestrul 4/MPP/lab03/java/src/main/java/mpp/domain/Result.java
package mpp.domain;

public class Result extends Entity<Long>{

    private Event event;
    private Participant participant;
    private Discipline discipline;
    private int points;

    public Result(Long aLong, Event event, Participant participant, Discipline discipline, int points) {
        super(aLong);
        this.event = event;
        this.participant = participant;
        this.discipline = discipline;
        this.points = points;
    }

    public Result(Event event, Participant participant, Discipline discipline, int points) {
        this.event = event;
        this.participant = participant;
        this.discipline = discipline;
        this.points = points;
    }

    public Event getEvent() {
        return event;
    }

    public void setEvent(Event event) {
        this.event = event;
    }

    public Participant getParticipant() {
        return participant;
    }

    public void setParticipant(Participant participant) {
        this.participant = participant;
    }

    public Discipline getDiscipline() {
        return discipline;
    }
    public void setDiscipline(Discipline discipline) {
        this.discipline = discipline;
    }
    public int getPoints() {
        return points;
    }
    public void setPoints(int points) {
        this.points = points;
    }
}

File: Semestrul 4/SDI/lab04/cs/Domain/Result.cs
namespace cs.Domain;

public class Result : IEntity<long>
{
    private Event @event;
    private Participant participant;
    private Discipline discipline;
    private int points;

    public Result(long id, Event @event, Participant participant, Discipline discipline, int points)
        : base(id)
    {
        this.@event = @event;
        this.participant = participant;
        this.discipline = discipline;
        this.points = points;
    }

    public Result(Event @event, Participant participant, Discipline discipline, int points)
    {
        this.@event = @event;
        this.participant = participant;
        this.discipline = discipline;
        this.points = points;
    }

    public Event GetEvent()
    {
        return @event;
    }

    public void SetEvent(Event @event)
    {
        this.@event = @event;
    }

    public Participant GetParticipant()
    {
        return participant;
    }

    public void SetParticipant(Participant participant)
    {
        this.participant = participant;
    }

    public Discipline GetDiscipline()
    {
        return discipline;
    }

    public void SetDiscipline(Discipline discipline)
    {
        this.discipline = discipline;
    }

    public int GetPoints()
    {
        return points;
    }

    public void SetPoints(int points)
    {
        this.points = points;
    }
}